.github/workflows/ci.yaml

```diff
@@ -21,14 +22,26 @@ jobs:
         with:
           python-version: "3.10"
 
-      - name: Install poetry
-        uses: abatilo/actions-poetry@v3
+      - name: Install pipx
```

Comment:
good catch on this! The `abatilo/actions-poetry@v3` claims to already install with pipx, wonder why it was breaking. Maybe something to do with `$GITHUB_PATH` variable? could be worth opening an issue for the maintainer if there isn't one already